### PR TITLE
[T2A5][T10-A4]ZavierOng

### PR DESCRIPTION
**Pull request by Zavier-Ong**

### CODE REVIEW

**Comment by Zavier-Ong** [comment]
Ready for review

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -357,6 +362,29 @@ public static String executeCommand(String userInputString) {
...
     }
+    
+    /**
+    * Displays and sorts all persons in the address book to the user; in alphabetical order.
+    * 
+    * @return feedback display message for the operation result
+    */
+   private static String executeSortAllPersonsInAddressBook() {
+       sortAddressBook();
+       return executeListAllPersonsInAddressBook();
```
Comment (mauris) [inline src/seedu/addressbook/AddressBook.java:373]: good job reusing existing code and ensuring SLAP

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -421,10 +449,25 @@ private static String getMessageForSuccessfulAddPerson(String[] addedPerson) {
...
+    /**
+     * Changes string to lower case to cater to case insensitivity
+     * 
+     * @param keywords
+     * @return lower case keywords
+     */
+    private static Set<String> changeToLowerCase(Collection<String> keywords) {
+      	Set<String> lowerCaseStrings = new HashSet<String>();
+    	for (String i: keywords){
+    		lowerCaseStrings.add(i.toLowerCase());
```
Comment (mauris) [inline src/seedu/addressbook/AddressBook.java:467]: do take note of indentation

**Comment by mauris** [comment]
Good job completing task 5. keep it up!

**Pull request closed without merging**